Automattic/wp-calypso
# Site setup: Add task for updating menus

#### Changes proposed in this Pull Request

- Add new task to the site setup checklist for updating the menus of the site.
- Task opens a dialog with the Site Menus support article (where a video will be added in the future).
- Add support for a custom button label and URL in the support article dialogs.
- Dialog links to the Customizer's menus section.
- Display skip button as a link.
- Removes the "You completed a task!" notice when skipping a task.

<img width="647" alt="Screen Shot 2020-04-08 at 11 24 13" src="https://user-images.githubusercontent.com/1233880/78767921-7fd06400-798b-11ea-8fe7-b382480b23ca.png">

<img width="1081" alt="Screen Shot 2020-04-07 at 14 42 02" src="https://user-images.githubusercontent.com/1233880/78670410-fdd63180-78dd-11ea-95dc-c727cfa5715e.png">

#### Testing instructions

- Load Calypso running this branch and switch to a simple site with the checklist incomplete.
- Make sure there is a task for updating the menus (it should be the 2nd one if the site title has not been set yet).
- Make sure clicking on "Start with a tutorial" opens a dialog with the content of https://wordpress.com/support/menus/.
- Go to the Customizer and update the site menu.
- Go back to My Home and make sure the task has been completed.

Fixes #40345
